pkg/config/config.go

```diff
@@ -1675,22 +1673,22 @@ func (r *RuntimeHandler) ValidateNoSyncLog() error {
 	return fmt.Errorf("no_sync_log is only allowed with runtime type 'oci', runtime type is '%s'", r.RuntimeType)
 }
 
-// SetContainerMinMemory sets the minimum container memory for a given runtime.
+// ValidateContainerMinMemory sets the minimum container memory for a given runtime.
 // assigns defaultContainerMinMemory if no container_min_memory provided.
-func (r *RuntimeHandler) SetContainerMinMemory() (int64, error) {
+func (r *RuntimeHandler) ValidateContainerMinMemory(name string) error {
 	if r.ContainerMinMemory == "" {
 		r.ContainerMinMemory = units.BytesSize(defaultContainerMinMemory)
 	}
 
-	memoryBytes, err := units.RAMInBytes(r.ContainerMinMemory)
+	memorySize, err := units.RAMInBytes(r.ContainerMinMemory)
 	if err != nil {
-		err = fmt.Errorf("unable to set runtime memory to %q: %w", r.ContainerMinMemory, err)
+		err = fmt.Errorf("unable to set runtime memory to %q: %w. Setting to %q instead", r.ContainerMinMemory, err, defaultContainerMinMemory)
```

Comment:
Nit. Just to keep errors in lower case in spirit.

```suggestion
		err = fmt.Errorf("unable to set runtime memory to %q: %w; setting to %q instead", r.ContainerMinMemory, err, defaultContainerMinMemory)
```